specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2020-02-14-privatepreview/postgresql.json

```diff
@@ -986,21 +877,24 @@
               "$ref": "#/definitions/CloudError"
             }
           }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.DBForPostgreSql/checkNameAvailability": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DBForPostgreSql/locations/{locationName}/checkVirtualNetworkSubnetUsage": {
       "post": {
         "tags": [
-          "CheckNameAvailability"
+          "VirtualNetworkSubnetUsage"
         ],
-        "operationId": "CheckNameAvailability_Execute",
+        "operationId": "VirtualNetworkSubnetUsage",
```

Comment:
Follow convention, this needs to be 'VirtualNetworkSubnetUsage_Execute'

Reply:
Done